src/modules/UI/scenes/Settings/action.js

```diff
@@ -167,13 +168,11 @@ export const setBitcoinOverrideServerRequest = (overrideServer: string) => (disp
 
 // touch id interaction
 export const updateTouchIdEnabled = (arg: boolean, account: EdgeAccount) => async (dispatch: Dispatch, getState: GetState) => {
-  const context = CORE_SELECTORS.getContext(getState())
-  // dispatch the update for the new state for
   dispatch(SETTINGS_ACTIONS.updateTouchIdEnabled(arg))
   if (arg) {
-    enableTouchId(context, account)
+    enableTouchId(makeReactNativeFolder(), account)
```

Comment:
Did this API change? If so this is a breaking change and the major version should bump accordingly

Reply:
Yes it did. we removed context.io.folder I will republish Login Ui and do the version accordingly.